jira-releasenotes/src/jira-releasenotes.js

```diff
@@ -78,9 +78,9 @@ const createReleaseNotes = async ({
         }
         core.info(`Skip issue ${issueKey}. It already has a release note`);
         return null;
-      }));
+      })
+      .catch((err) => core.warning(`Failed to update issue. ${err.message}`)));
```

Comment:
If I'm not reading this in the wrong way, then I believe the catch clause is misplaced. You want to append it right after the `.then(..)` block. Now I believe it is chained to the `push(..)` call.